subsys/bluetooth/shell/l2cap.c

```diff
@@ -305,15 +372,49 @@ static int cmd_metrics(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_whitelist_add(const struct shell *shell, size_t argc, char *argv[])
+{
+	int i;
+
+	if (!default_conn) {
+		printk("Not connected\n");
```

Comment:
printk is not recommened within command context as it may compete with shell or logger printing functions.
I recommend to check it to `shell_fprintf(shell, SHELL_ERROR, "Not connected\n");`

Reply:
or you can reuse macro `error`

Reply:
done